README.md

```diff
@@ -262,6 +171,28 @@ helloworld-2559879000-XXXXXX  myResourceGroup    Succeeded            microsoft/
 ```
 </details><br/>
 
+### Pulling images using user assigned managed identity
```

Comment:
Would you please follow the same as the upgrade, etc docs? by creating a new file in the /docs folder and referring to it in this main one?